x-pack/plugins/alerting/public/pages/maintenance_windows/helpers/convert_from_maintenance_window_to_form.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment';
+import { has } from 'lodash';
+import { MaintenanceWindow } from '../types';
+import { EndsOptions, Frequency } from '../constants';
+import { FormProps, RecurringScheduleFormProps } from '../components/schema';
+import { getInitialByWeekday } from './get_initial_by_weekday';
+import { RRuleParams } from '../../../../common';
+
+export const convertFromMaintenanceWindowToForm = (
+  maintenanceWindow: MaintenanceWindow
+): FormProps => {
+  const startDate = maintenanceWindow.rRule.dtstart;
+  const endDate = moment(startDate).add(maintenanceWindow.duration);
+  // maintenance window is considered recurring if interval is defined
+  const recurring = has(maintenanceWindow, 'rRule.interval');
+  const form: FormProps = {
+    title: maintenanceWindow.title,
+    startDate,
+    endDate: endDate.toISOString(),
+    recurring,
+  };
+  if (!recurring) return form;
+
+  if (recurring) {
```

Comment:
this `if` shouldn't be needed, given the **not** version is just above it.

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/154669/commits/b2745dcd188cc86ffc5ecffcfc724d66cc13803f